feature-pack-build-maven-plugin/src/main/java/org/wildfly/build/plugin/FeaturePackBuildMojo.java

```diff
@@ -65,13 +65,13 @@
     /**
      * The configuration file used for feature pack.
      */
-    @Parameter(alias = "config-file", defaultValue = "feature-pack-build.xml", property = "wildfly.feature.pack.configFile")
+    @Parameter(alias = "config-file", defaultValue = "${basedir}/feature-pack-build.xml", property = "wildfly.feature.pack.configFile")
     private String configFile;
```

Comment:
I think we need a new config param. As you say maybe nobody has modified it from the default but we have no way to be sure.